srcpkgs/corosync/update

```diff
@@ -0,0 +1 @@
+site="https://github.com/corosync/corosync/releases"
```

Comment:
Remove this file after making the github url change above; the target is automatically determined for compliant distribution URLs.